src/main.py

```
@@ -1,232 +1,161 @@
 import sys
 import os
+import random
+import time
+import wx
+import wx.adv
+import subprocess
+import wx.stc as stc
```

Pylint (3.3.7)

[refactor] Use 'from wx import stc' instead

(R0402)